src/datetime.rs

```diff
@@ -53,11 +49,11 @@ impl fmt::Display for DateTime {
             crate::display_num_buf(2, 17, self.time.second as u32, &mut buf);
             f.write_str(std::str::from_utf8(&buf[..]).unwrap())?;
         }
-        if let Some(offset) = self.offset {
-            if offset == 0 {
+        if let Some(tz_offset) = self.time.tz_offset {
```

Comment:
can we reuse this logic from `time.rs`?

Reply:
I have used the `Date::fmt` and `Time::fmt` instead. There's an overhead for the function call I think.